oak_enclave_runtime_support/src/heap.rs

```diff
@@ -79,14 +79,23 @@ impl GrowableHeap {
         }
     }
 
+    /// # Safety
+    ///
+    /// Upheld by caller.
     pub unsafe fn init(&mut self) {}
```

Comment:
@andrisaar since this is now public clippy asked me to fix a couple of rule violations. I'm unsure why this function exists. Given that it does nothing, a meaningful safety comment seemed difficult. 

Does this function exist by convention only?

Reply:
Good question, I don't remember. Feel free to get rid of it.